common/scala/src/main/scala/whisk/connector/kafka/KafkaConsumerConnector.scala

```diff
@@ -99,9 +84,10 @@ class KafkaConsumerConnector(
         val props = new Properties
         props.put(ConsumerConfig.GROUP_ID_CONFIG, groupid)
         props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkahost)
-        props.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, "10000");
+        props.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, 10000.toString)
         props.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, true.toString)
-        props.put(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG, "10000");
+        props.put(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG, 10000.toString)
+        props.put(ConsumerConfig.MAX_POLL_RECORDS_CONFIG, maxMessagePerPoll.toString)
```

Comment:
using kafka 10 feature to provide a bound on max records returned from poll per partition.
default set to preserve existing behavior (return all records).